assets/js/logging.js

```diff
@@ -0,0 +1,21 @@
+var colors = require('colors');
```

Comment:
I'm not sure I understand it. What do you expect will happen with these unpacked assets? Are they auto-generated? Are they to be distributed? If so, remember that the Python package is the main container, and it will not allow data files outside of the package!